types/sortablejs/index.d.ts

```diff
@@ -255,7 +332,7 @@ declare namespace Sortable {
         /**
          * Event when you move an item in the list or between lists
          */
-        onMove?: (event: MoveEvent) => boolean;
+        onMove?: (evt: MoveEvent, originalEvent: Event) => void;
```

Comment:
This return type shouldn't be `void`. The docs say:

```javascript
		// return false; — for cancel
		// return -1; — insert before target
		// return 1; — insert after target
```

Though, reading the actual code, it seems returning `true` would also work. So maybe `=> boolean | -1 | 1`?

Reply:
Yes, that is correct.

I will change this. That was a mistake on my part.